src/utils/theme.ts

```diff
@@ -24,6 +24,15 @@ const base: ThemeOptions = {
     typography: {
         fontFamily: getFontFamily(),
     },
+    breakpoints: {
+        values: {
+            xs: 0,
+            sm: 600,
+            md: 1112,
```

Comment:
According to https://developer.apple.com/library/archive/documentation/DeviceInformation/Reference/iOSDeviceCompatibility/Displays/Displays.html, the largest width of iPad is 1112.